src/index.ts

```diff
@@ -9,9 +9,8 @@ const repo = context.payload.repository.name;
 
 const run = async () => {
   console.log("Starting release changelog");
-  const version =
-    core.getInput("version") || context.ref.replace("refs/tags/", "");
-  const githubApiKey = process.env.GITHUB_TOKEN;
+  const version = core.getInput("version") || context.ref.replace("refs/tags/", "");
+  const githubApiKey = core.getInput("GITHUB_TOKEN");
```

Comment:
Bør følge samme navngivning som resten av inputs.
Også bør den fortsatt støtte miljøvariabel, så slipper vi å gjøre den til en 2.0 release
```suggestion
  const githubApiKey = core.getInput("github-token") || process.env.GITHUB_TOKEN;
```

Reply:
ja, enig i at den bør støtte process.env.

Eneste grunn til jeg liker `GITHUB_TOKEN` er mest fordi da er det tydelig hva vi snakker om, ettersom det er det samme som `secrets.GITHUB_TOKEN`